lib/checks/color/color-contrast-evaluate.js

```diff
@@ -150,7 +156,8 @@ function colorContrastEvaluate(node, options, virtualNode) {
     fontSize: `${((fontSize * 72) / 96).toFixed(1)}pt (${fontSize}px)`,
     fontWeight: bold ? 'bold' : 'normal',
     messageKey: missing,
-    expectedContrastRatio: expected + ':1'
+    expectedContrastRatio: expected + ':1',
+    shadowColor: shadowColors.map(c => c.toHexString()).join(', ')
```

Comment:
I think we should put the flattened shadow color in here. I thought I left that comment before, but can't seem to find it.

Reply:
Made that change, but this will result in some strange seeming messages - 
```
  <div
    id="target"
    style="background-color: #aaa; color:#666; text-shadow: 1px 1px 0.01em #000"
  >
    Hello world
  </div>
  ```
  will report that the text shadow color is #aaa when the user would reasonably assume that the color should be #000. Do we need to update the failure messages somehow?


  ```
  Fix any of the following:\n  Element has insufficient color contrast of 2.47 between the foreground and shadow color (foreground color: #666666, text-shadow color: #aaaaaa, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
  ```